src/autopas/pairwiseFunctors/CellFunctor.h

```diff
@@ -151,7 +151,7 @@ void CellFunctor<ParticleCell, ParticleFunctor, bidirectional>::processCell(Part
     return;
   }
 
-  switch (_dataLayout) {
+  switch (static_cast<DataLayoutOption::Value>(_dataLayout)) {
```

Comment:
```suggestion
  // Static cast only needed for Apple Clang
  switch (static_cast<DataLayoutOption::Value>(_dataLayout)) {
```
Add a comment like this to all four instances and add the version with which you encountered the compiler bug.

Reply:
Comments added ✅